ui/admin/app/components/form/worker/index.hbs

```diff
@@ -0,0 +1,85 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+
+  {{#unless form.isEditable}}
+    <form.input
+      @name='type'
+      @value={{@model.type}}
+      @label={{t 'form.type.label'}}
+      @helperText={{t 'form.worker_type.help'}}
+      readonly={{true}}
+    />
+  {{/unless}}
+
+  <form.input
+    @name='name'
+    @type='name'
+    @value={{@model.name}}
+    @label={{t 'form.name.label'}}
+    @helperText={{t 'form.name.help'}}
+    @error={{@model.errors.name}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.name}}
+      <field.errors as |errors|>
+        {{#each @model.errors.name as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.input>
+
+  <form.textarea
+    @name='description'
+    @type='description'
```

Comment:
I don't think there's a `description` type. I also don't think the `textarea` component takes in a `@type` so this can probably be removed.